### PR TITLE
Build Diona Different

### PR DESCRIPTION
**Pull request by Lyndomen**
## About the PR
Somewhat restores Nyano diona, they are slower & more tanky, and cooler. 

## Why / Balance
Janitor diona is the worst experience

- Slows sprint 4.5 > 3
- Slows Walk 2.5 > 1, notably this means they can walk on space lube and not slip. 

- Piercing:  1 > 0.6
- Blunt: .7 > .6
- Slash:  .8 > .7

Changes Ignite threshold (max heat damage taken before ignition)
12 > 20. A diona with zero armor takes 21 heat from a normal laser and will still ignite. 10% heat resistance will put them under that threshold- or a bigger laser. 

## Technical details
only yaml

## Media
image
my video was too big but they slow

## Requirements
- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**
:cl:
- tweak: Diona are substantially more tanky to physical damage, and ignite less frequently.
- tweak: Diona are much slower, but cannot slip while walking.

### CODE REVIEW

**Comment by Bonktrauma** [comment]
FINALLY!!! THE RETUrN TO OUR GLORY DAYS OF SLOW DIONA

**Comment by CybersunOperative** [comment]
Hi. Direction here. Seems fine from a balance perspective. Heat damage ignite threshold has probably been long overdue for an increase. I'm sure our three total Diona players will enjoy being able to RP their epic slowness again.

Other voices in direction have said that they support this PR as well. :3

**Comment by Bonktrauma** [comment]
> Hi. Direction here. Seems fine from a balance perspective. Heat damage ignite threshold has probably been long overdue for an increase. I'm sure our three total Diona players will enjoy being able to RP their epic slowness again

This is hilariously accurate,  I can go like 3 shifts without seeing a single other diona player

File: Resources/Prototypes/Entities/Mobs/Species/diona.yml [hunk 1]
```diff
@@ -110,9 +110,12 @@
           32:
             sprite: Mobs/Species/Human/displacement.rsi
             state: jumpsuit-female
-  - type: FootPrints # DeltaV port from EE, blood splatter
+  - type: FootPrints # begin DeltaV
```
Comment (MilonPL) [inline Resources/Prototypes/Entities/Mobs/Species/diona.yml]: should really just be two separate comments, ond on the footprints and one on the movement modifier comp

**Review by MilonPL — CHANGES_REQUESTED** [review]

**Review by MilonPL — APPROVED** [review]

**Pull request merged**